src/languages/en.ts

```diff
@@ -3140,7 +3140,7 @@ const translations = {
             brokenConnectionErrorFirstPart: `Card feed connection is broken. Please `,
             brokenConnectionErrorLink: 'log into your bank ',
             brokenConnectionErrorSecondPart: 'so we can establish the connection again.',
-            assignedYouCard: ({link}: AssignedYouCardParams) => `assigned you a ${link}! Imported transactions will appear in this chat.`,
```

Comment:
You can check how it will look at the attached to this PR screenshots, example:
![image](https://github.com/user-attachments/assets/808541f9-f746-48be-8660-513375a9dffd)

More context can be found here: https://callstack-hq.slack.com/archives/C06ML6X0W9L/p1729769612494509?thread_ts=1729596891.077979&cid=C06ML6X0W9L